scripts/build-container.sh

```diff
@@ -4,10 +4,11 @@
 OLS_VERSION=v0.0.0
 
 # To build container for local use
-podman build --no-cache --build-arg=VERSION=${OLS_VERSION} -t ols:latest -f Containerfile
-
-# To publish to a registry tag registry name
-# podman tag ols:latest quay.io/<org>/ols:$OLS_VERSION
```

Comment:
tagging step not needed any more because you can just use OLS_API_IMAGE to build/tag the image with the appropriate name in one shot.

Reply:
Okay. It is commented out so it can be removed.

/lgtm